src/main/java/hudson/plugins/templateproject/ProxyBuilder.java

```diff
@@ -45,16 +34,22 @@ public String getProjectName() {
 		return projectName;
 	}
 
-	public String getExpandedProjectName(AbstractBuild<?, ?> build) {
```

Comment:
I have a suggestion that this method was declared as public accidentally. I don't see any particular reason to expose it as public. There are no usages in this project, but I'm not sure about other plugins. Is there a way to find what plugins depends on this plugin?